src/test/shell/bazel/cpp_darwin_integration_test.sh

```diff
@@ -96,5 +96,22 @@ EOF
       fail "Stripping failed, debug symbols still found in the stripped binary"
 }
 
+# Regression test for https://github.com/bazelbuild/bazel/pull/12046
+function test_osx_sandboxed_cc_library_build() {
+  mkdir -p cpp/osx_sandboxed_cc_library_build
+  cat > cpp/osx_sandboxed_cc_library_build/BUILD <<EOF
+cc_library(
+    name = "a",
+    srcs = ["a.cc"],
+)
+EOF
+  cat > cpp/osx_sandboxed_cc_library_build/a.cc <<EOF
+void a() { }
+EOF
+  assert_build --spawn_strategy=sandboxed //cpp/osx_sandboxed_cc_library_build:a
```

Comment:
Is it possible to verify here that the sandbox was really used for this action?

I'm concerned about --spawn_strategy being overridden by some other --strategy flag and not having the right effect for this test.

Reply:
I don't know how to check what strategy is used for the action, but we can check in BEP that no other strategy flag was used.